src/prefect/flows.py

```diff
@@ -798,6 +798,7 @@ def serve(
         rrule: Optional[Union[Iterable[str], str]] = None,
         paused: Optional[bool] = None,
         schedules: Optional["FlexibleScheduleList"] = None,
+        global_deployment_limit: Optional[int] = None,
```

Comment:
I think `global_limit` is preferable for brevity.
```suggestion
        global_limit: Optional[int] = None,
```